e2e/specs/st_image.spec.js

```diff
@@ -91,4 +91,16 @@ describe("st.image", () => {
         .matchImageSnapshot("black-square-column");
     }
   });
+
+  it("displays SVG images that load external images", () => {
+    cy.get("[data-testid='stImage'] svg")
+      .eq(0)
+      .matchImageSnapshot("karriebear-avatar");
+  });
+
+  it("displays links in text as text", () => {
+    cy.get("[data-testid='stImage'] svg")
+      .eq(1)
+      .should("contain", "avatars.githubusercontent");
+  });
```

Comment:
Assuming it would be relatively easy to construct, could you add a test case that verifies that SVG sanitization is actually happening? Like, an SVG test string that will log to the console or open a modal or something else detectable - and a test that asserts that code doesn't actually get run?

(The test could either go in here, or possibly just in `ImageList.test.tsx` to prevent having to do all the e2e machinery.)

Reply:
I'm just popping in to point out how I am now part of our codebase 😈

Reply:
Good idea, I'll write that now

Reply:
It was actually not easy to construct, at least in a form that would make me confident my test could catch an issue if we didn't do sanitization, because either I'm writing the code wrong, or because the automatic partial jsx sanitization is catching it. Either way, I don't want to include a test for something if it will give unwarranted confidence that it would alert us if something started going wrong.